zkevm-circuits/Cargo.toml

```diff
@@ -34,6 +34,7 @@ integer =   { git = "https://github.com/privacy-scaling-explorations/halo2wrong"
 libsecp256k1 = "0.7"
 num-bigint = { version = "0.4" }
 subtle = "2.4"
+rand_chacha = "0.3"
```

Comment:
This is related regarding my common on Rng

Reply:
At the moment, it also seems we just using it to mock stuff. If so, we can drop chacha completely and use `StdRng` instead.

Reply:
Marking this as resolved as for now we're using a "constant" point generated with a deterministic rng.